lua/holohud2/modules/util.lua

```diff
@@ -359,9 +359,9 @@ hook.Add( "PreDrawHUD", "holohud2_util", function()
             
             end
 
-            primary = ( ammo.PrimaryClip or ammo.PrimaryAmmo ) and 1 or 0
+            primary = weapon:GetPrimaryAmmoType()
```

Comment:
Previous behavior seemed incorrect and resulted in TacRP weapons (TacRP implements CustomAmmoDisplay so it is affected) not showing the correct ammo type.